oscar_elasticsearch/search/__init__.py

```diff
@@ -1 +1 @@
-default_app_config = "oscar_elasticsearch.search.config.SearchConfig"
```

Comment:
To keep 1.6 compatibility, this need to remain unchanged. Instead, we can specify in the readme to intall the search app from the appcinfig directly:

"oscar_elasticsearch.search.apps. OscarElasticSearchConfig"

Reply:
Ok done.